frame/election-provider-multi-phase/src/mock.rs

```diff
@@ -241,8 +241,9 @@ impl pallet_balances::Config for Runtime {
 	type WeightInfo = ();
 }
 
-#[derive(Eq, PartialEq, Debug, Clone, Copy)]
```

Comment:
Why is this needed?

Reply:
`fn take` from parameter_types needs a default at compile time.